requests.md

```diff
@@ -0,0 +1,25 @@
+#Tech Requests
+
+To ensure that DxE Tech is able to swiftly help you with your tech request, we need the following information:
+
+* Summary of the problem or request
+  * For tech problems, please include any error messages and screenshots of the issue(s), how long the problem has been happening or when it was discovered, and whom the issue impacts
```

Comment:
Maybe bold "tech problems" and "tech requests" to make the structure of this section more obvious?